common/djangoapps/enrollment/views.py

```diff
@@ -581,8 +586,19 @@ def post(self, request):
                 )
 
             enterprise_course_consent = request.data.get('enterprise_course_consent')
```

Comment:
We should add a TODO below explaining that this code will need to be refactored once we roll out both these changes and the ecommerce changes. We should also create a follow ticket for doing this work.